src/kili/services/asset_import/legacy.py

```diff
@@ -179,27 +180,38 @@ def upload_content(signed_url: str, content: str, input_type: str):
         raise ValueError(f"File: {content} not found")
 
 
+def generate_unique_ids(size: int) -> List[str]:
+    """
+    Returns ids
+    """
+    return [cuid.cuid() for _ in range(size)]
+
+
 # pylint: disable=too-many-arguments
 def process_and_store_content(
-    input_type: str,
+    legacy_data_importer: LegacyDataImporter,
```

Comment:
passing the full `LegacyDataImporter` is maybe too much (passing two arguments like `input_type` and `Auth`) is ok + it's legacy. @theodu wdyt?

Reply:
it's legacy anyway, so I don't want to bother myself with adding multiple parameters